.travis.yml

```diff
@@ -47,15 +48,22 @@ matrix:
     - os: linux
       compiler: gcc
       env: DISTRO=centos DISTRO_VERSION=7 BUILD_TYPE=Release
+    - # This is a fairly specialized entry in the matrix.  It builds the clients
```

Comment:
Do you want to only run this on the `master` branch, and not any of the PRs, to speed up the review / merge process? Or is the goal to run this on every PR (and every PR revision)?

Reply:
Good point.  Let's skip it for PRs.  Too slow, like the macOS build.